ui-components/src/blocks/ImageCard.tsx

```diff
@@ -28,9 +32,13 @@ const ImageCard = (props: ImageCardProps) => {
     <div className="image-card__wrapper">
       <figure className="image-card">
         {props.imageUrl && (
-          <img src={props.imageUrl} alt={props.description || "A picture of the building"} />
+          <img src={props.imageUrl} alt={props.description || t("listings.buildingImageAltText")} />
+        )}
+        {props.listing?.reservedCommunityType && (
+          <Tag styleType={AppearanceStyleType.warning} className={"image-card__tag"}>
+            {t(`listings.reservedCommunityTypes.${props.listing.reservedCommunityType.name}`)}
+          </Tag>
         )}
```

Comment:
@emilyjablonski ,
There are efforts in #1555 to generalize ImageCard and adds a tag, so I'd like to coordinate with @akegan and @aekong on this. I'm not sure I have a preference on how the tags are handled but I would like this to be based off a prop that has nothing to do with listings and reserved community types.

Reply:
Can do 👍  Reviewing now, let's get the other one in first and I'll adjust from here

Reply:
Rebased from the other PR, so going to merge here :)